packages/cardano-services/src/ChainHistory/DbSyncChainHistory/types.ts

```diff
@@ -102,6 +102,24 @@ export interface RedeemerModel {
   tx_id: Buffer;
 }
 
+type VoterRole = 'ConstitutionalCommittee' | 'DRep' | 'SPO';
+
+export interface VotingProceduresModel {
+  tx_id: Buffer;
+  voter_role: VoterRole;
+  drep_voter: Buffer | null;
+  committee_voter: Buffer | null;
+  pool_voter: Buffer | null;
+  governance_action_tx_id: Buffer;
+  governance_action_index: number;
+  drep_hash_raw: string;
```

Comment:
```suggestion
  drep_hash_view: string;
```
to match the suggested fetching of bech32 data from db directly